R/tidyeval-across.R

```diff
@@ -132,16 +139,28 @@ across_setup <- function(data,
     names_vars <- names(tbl)[locs]
   }
 
-  dots <- lapply(call$..., partial_eval, data = data, env = env)
-  names_spec <- eval(call$.names, env)
-  funs_across_data <- across_funs(
-    funs = call$.fns,
-    env = env,
-    data = data,
-    dots = dots,
-    names_spec = names_spec,
-    fn = fn
-  )
+  try_fetch({
```

Comment:
It'd be slightly nicer to flip the `lapply()` and the `try_fetch()` so that the error message could reference the specific element of `...` that failed. (If you did this, for simplicity, I'd suggest coverting the `lapply()` to a for loop)